webapp/channels/src/components/dynamic_virtualized_list/item_row.tsx

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import debounce from 'lodash/debounce';
+import type {ReactNode} from 'react';
+import React, {memo, useEffect, useRef} from 'react';
+
+const RESIZE_DEBOUNCE_TIME = 120; // in ms
```

Comment:
How did you come up with this number? Can you add some context in the comment to capture the decision making process?

Reply:
Its the usual debounce time for UI interactions like scroll events